packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/class_details.dart

```diff
@@ -137,69 +127,39 @@ class _RetainedSizeColumn extends ColumnData<StatsByPathEntry> {
       )!;
 }
 
-class _RetainingPathTable extends StatefulWidget {
+class _RetainingPathTable extends StatelessWidget {
   _RetainingPathTable({
     Key? key,
     required this.data,
-    required this.sorting,
     required this.controller,
     required this.item,
-  })  : _columns = _tableColumns(sorting),
-        super(key: key);
-
-  final ClassStats data;
-  final ColumnSorting sorting;
-  final SnapshotInstanceItem item;
-  final DiffPaneController controller;
-
-  final List<ColumnData<StatsByPathEntry>> _columns;
-
-  static List<ColumnData<StatsByPathEntry>> _tableColumns(
-    ColumnSorting sorting,
-  ) {
-    final _shallowSizeColumn = _ShallowSizeColumn();
-
-    final result = <ColumnData<StatsByPathEntry>>[
+  }) : super(key: key) {
+    _columns = <ColumnData<StatsByPathEntry>>[
```

Comment:
make this a static const so that _columns is not reinitialized each time this widget is re created